src/scene/main_scene.cpp

```diff
@@ -36,40 +36,41 @@ void MainScene::setup_scene() {
     light_pos = glm::vec3(0.f, 2.f, 1.f);
 
     //Setting up scene graph for Grid
-    grid = new Grid(20, 20);
+    grid = new Grid("assets/maps/dungeon_test");    
```

Comment:
Also, if we're going with our custom data format, would prefer to have some file extension on this (`.txt` at the least)